src/notebook/cells/widget.ts

```diff
@@ -125,7 +125,22 @@ class BaseCellWidget extends Widget {
     this.layout = new PanelLayout();
 
     let renderer = options.renderer || BaseCellWidget.defaultRenderer;
-    this._editor = renderer.createCellEditor(null);
+    this._editor = renderer.createCellEditor({
+      indentUnit: 4,
+      readOnly: false,
+      theme: 'default',
+      extraKeys: {
+        'Cmd-Right': 'goLineRight',
+        'End': 'goLineRight',
+        'Cmd-Left': 'goLineLeft',
+        'Tab': 'indentMore',
```

Comment:
Thank you! I just got hit with this today when I was trying to indent a block of code, and all it kept doing was pulling up the completion widget!